src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
take a look at the [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md) pls and fix your layout accordingly

Reply:
I don't know you, Nazar, but I already worship your patience. 
Thanks!